src/Config.js

```diff
@@ -41,7 +41,13 @@ class Config {
   }
 
   static get sinopiaEnv() {
-    return process.env.SINOPIA_ENV || '' // default to production, which requires no additional display
+    if (process.env.SINOPIA_ENV) {
```

Comment:
What's up above should be equivalent to this

Reply:
yeah, it is now, bit I'm going to add in a ' - ' to the set value for display purposes (instead of doing that conditional in the view component code)

Reply:
oh dang, need my :eyeglasses: